### PR TITLE
Fixes #2750 - Improves text rendering

### LINKED ISSUE #2750: Enable text compression
I was running some profilers and PageSpeed gave me this result:
screen shot 2018-12-28 at 10 53 10
 
I've already wrote a little bit about how to do that on my [proposal](https://psychonautgirl.space/webcompat.html#Future-implementations-in-details). It might be a good thing to start from these sources.

If PageSpeed is telling the true than we can get huge improvements for mobile users. As big as the ones removing markdown-it will do.

This is somewhat related to #2747 because there are some alternatives of doing this on the server side.

### PR DESCRIPTION
**Pull request by marimeireles**
This PR follows the previous discussion we had here: https://github.com/webcompat/webcompat.com/pull/2808
I decided to create a new version so we still keep a log from the discussion we had about trying to use the `fontfaceobserver`.

There are two changes in this PR.
1. I removed the PT+Mono webfont because Mike and I agreed it wasn't worth it to make a request for that font. @karlcow what do you think?
2. Added some configuration to our fonts using `@font-face`. The reason why I added it can be found in this https://github.com/webcompat/webcompat.com/issues/2750#issuecomment-464926948 but summarizing I think the `font-display: optional` option can have a huge impact for users in slow internet connections showing text for them much earlier + this is super easy to implement.

@miketaylr these are the changes I proposed here https://github.com/webcompat/webcompat.com/pull/2808#issuecomment-473409855.

### CODE REVIEW

**Comment by marimeireles** [comment]
wow this is weird. Why my PR isn't being tested?

**Comment by karlcow** [comment]
> wow this is weird. Why my PR isn't being tested?

indeed.

File: webcompat/templates/layout.html [hunk 1]
```diff
@@ -46,3 +46,4 @@
 {% block extrascripts %}{% endblock %}
 </body>
 </html>
+</html>
```
Comment (karlcow) [inline webcompat/templates/layout.html]: Hmm this doesn't seem to be a legit thing 😜

Comment (marimeireles) [reply webcompat/templates/layout.html]: hehe
Sorry for that!

**Comment by marimeireles** [comment]
@karlcow 🐄

**Comment by karlcow** [comment]
yoohoo @marimeireles Hello and welcome back.

**Pull request merged**